tools/osx/crosstool/cc_toolchain_config.bzl

```diff
@@ -55,41 +55,48 @@ def _deterministic_libtool_flags(ctx):
         return ["-D"]
     return []
 
+def _target_os_version(ctx):
+    platform_type = ctx.fragments.apple.single_arch_platform.platform_type
+    xcode_config = ctx.attr._xcode_config[apple_common.XcodeVersionConfig]
+    return xcode_config.minimum_os_for_platform_type(platform_type)
+
 def _impl(ctx):
+    target_os_version = _target_os_version(ctx)
```

Comment:
continuing the convo from the other PR, I guess I would assume this was only fetched at toolchain setup time, and thus wouldn't differ between targets?

Reply:
This is a normal rule, not a repo rule, so this value is evaluated at build time.

Reply:
I think the toolchain setup just symlinks this file to the `local_config_cc` repo.

Reply:
I have tested a Bazel binary built from this patch with all unit tests in rules_swift with the above mentioned rules_swift change.

Reply:
ah my bad